src/core/builtins.d

```diff
@@ -0,0 +1,13 @@
+/**********************************************
+ * This module implements common builtins for the D frontend.
+ *
+ * Copyright: Copyright © 2019, The D Language Foundation
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Walter Bright
+ * Source:    $(DRUNTIMESRC core/builtins.d)
+ */
+
+module core.builtins;
+
+/// Writes `s` to `stdout` during CTFE (does nothing at runtime).
```

Comment:
```suggestion
/// Writes `s` to `stderr` during CTFE (does nothing at runtime).
```

`pragma(msg)` writes to stderr, so I guess `__ctfeWrite` would so the same.

Reply:
That said, it may actually be a good idea to separate things, so that `pragma(msg)` and `__ctfeWrite` would use `stdio`, while all other compiler output (error, warning and deprecation messages) would still go to `stderr`.

Reply:
You can't ever assume that (stdout is used to emit compiled code for `-pipe` in gcc)

Reply:
The currently proposed implementation actually writes to `stdout`. I'll fix it.

Reply:
DMD is free to do what it wants so long as it's done in the non-common modules (builtins, compiler, ctfloat, errors, target, ...). At one point I introduced `stdmsg` to the compiler for just this reason (dmd and gdc emitted verbose and pragma messages to different streams).

Reply:
Nevertheless, it is better if the compilers are consistent in this regard (e.g. if tools want to use the output generated from `__ctfeWrite`).